assets/src/blocks/Covers/deprecated/coversV3.js

```diff
@@ -1,8 +1,15 @@
 import {frontendRendered} from '../../../functions/frontendRendered';
-import {attributes} from '../CoversBlock';
-import {BLOCK_NAME} from '../CoversConstants';
+import {ATTRIBUTES, BLOCK_NAME} from '../CoversConstants';
+
+const {__} = wp.i18n;
 
 export const coversV3 = {
-  attributes,
+  attributes: {
+    ...ATTRIBUTES,
+    readMoreText: {
+      type: 'string',
+      default: __('Load more', 'planet4-blocks'),
```

Comment:
Can we keep using "Read more"?
As we want to get rid of Cover blocks soon it doesn't make sense to introduce changes to the block.

Reply:
We'll need to make this change since it required by its spec.

Reply:
Do you mean the e2e test `tests/e2e/tools/lib/covers.js`?

Reply:
Yes

Reply:
To be honest, I'm not sure why it is failing the issue since the [released file is set with "Read More"](https://github.com/greenpeace/planet4-master-theme/blob/main/tests/e2e/tools/lib/covers.js#L109)

Reply:
I know what took another value. Fixing now! Thanks for following up on this 🙏

Reply:
FYI, is rolled back the change again, since we need this change to pass specs.